### PR TITLE
Release notes for apiURL change.

### PR DESCRIPTION
**Pull request by jkinkead**
Release notes for PR #2262 .

### CODE REVIEW

**Comment by typesafe-tools** [comment]
Can one of the admins verify this patch?

File: notes/0.13.10/always-honor-api-url.markdown [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[@jkinkead]: https://github.com/jkinkead
+[2262]: https://github.com/sbt/sbt/pull/2262
+
+### Fixes with compatibility implications
+
+### Improvements
+
+- The setting `apiURL` used to be ignored if `autoAPIMappings` was `false`. It will now be inserted into the built POM file regardless of the value of `autoAPIMappings`.  [#2262] by [@jkinkead]
```
Comment (Duhemm) [inline notes/0.13.10/always-honor-api-url.markdown]: It looks like the link to the PR does not show up as a link. In the other notes files it is done this way:

``` markdown
[#2262][2262]
```

**Comment by jkinkead** [comment]
Cargo cult fail. I don't usually use that style of markdown link - fixed now!

**Comment by Duhemm** [comment]
Great! LGTM

**Comment by dwijnand** [comment]
LGTM too. Thanks for following up on this.

**Pull request merged**